electionleaflets/apps/leaflets/tests/test_models.py

```diff
@@ -23,14 +24,22 @@ def test_markdown_error(self):
         leaflet = Leaflet.objects.create(title="Test Leaflet", description=None)
         response = self.client.get(leaflet.get_absolute_url())
         self.assertEqual(response.status_code, 200)
+        
+    def test_redirect_to_leaflet(self):
+        party = Party.objects.create(party_name="Labour Party") 
+        leaflet = Leaflet.objects.create(title="Test Leaflet", description=None, ynr_party_id="party:1", ynr_party_name="Labour Party", publisher_party=party)
+        response = self.client.get(leaflet.get_absolute_url())
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, "Test Leaflet")
+        self.assertContains(response, "Labour Party")
+    
 
-class LeafletImageTestCase(TestCase):
-    @skip("fix me after python and django upgrade")
```

Comment:
Is there a reason for removing this test, rather than re-enabling it?

Reply:
It's here https://github.com/DemocracyClub/electionleaflets/pull/257/commits/7c16355c16b1b46763790f5e1273e5a4ade5393b